_pytest/main.py

```diff
@@ -175,7 +175,7 @@ def pytest_ignore_collect(path, config):
     if excludeopt:
         ignore_paths.extend([py.path.local(x) for x in excludeopt])
 
-    if path in ignore_paths:
+    if py.path.local(path) in ignore_paths:
```

Comment:
@RonnyPfannschmidt / @nicoddemus 
You are right, it's already a py.path.local

It still fixes my issue because applying twice (?) it strips the leading `/`: 
```
(Pdb) path
local('///digest')
(Pdb) py.path.local(path)
local('/digest')
```

not sure where to fix that now